corehq/apps/fixtures/tests/test_field_names.py

```diff
@@ -154,10 +154,18 @@ def test_starts_with_number(self):
         bad_name = "0hello"
         self.assertTrue(is_identifier_invalid(bad_name))
 
-    def test_unicode(self):
+    def test_punctuation(self):
```

Comment:
this test was misleading as it wasn't the unicode causing issues, it was the punctuation